### ISSUE TITLE
[Editor] Showing/Hiding the editor and resetting the form value throws an error

### ISSUE THREAD

**Issue by PeterStaev** [telerik/kendo-angular#2908]
**Describe the bug**
When you hide & show the editor control and also reset the value of the bound form it throws the following error:
```
preview-6f2d8d3d3fde463d30fbb.js:1 ERROR TypeError: Cannot read property 'focusOffset' of null
    at selectionContext (index.js:417)
    at EditorView.updateStateInner (index.js:147)
    at EditorView.updateState (index.js:112)
    at EditorView.dispatchTransaction (main.js:5)
    at EditorView.dispatch (index.js:355)
    at eval (main.js:1)
    at e.exec (main.js:5)
    at e.set [as value] (main.js:5)
    at e.writeValue (main.js:5)
    at eval (shared.ts:113)
```

**To Reproduce**
https://stackblitz.com/edit/angular-xhyamj?file=app/app.component.ts
Click on Toggle 3 times to show, hide and then show again the editor and monitor the console. On the first show everything works ok. But for the second time it throws an error and the code after the form reset is not called. From what I see the value is correctly updated. 
Also I dont see such a behavior when NOT using forms but binding directly to the `value` property. 

**Expected behavior**
No error to be thrown

**Screenshots**

**Browser**
 - OS: Windows 10
 - Browser Chrome
 - Version 81

**Additional context**

**Comment by PeterStaev** [comment]
When can we expect a fix for this? Just got hit by this issue in a second application, and while I was able to hack around it in the first one, I can't do much in the second one...

**Comment by Alyyc85** [comment]
Experienced same issue, i've wrapped kendo-editor in another component providing NVA and implementing CVA.
In writeValue method i return if value is null or undefined.

It seems to work

**Comment by Celebrim42** [comment]
I'm impacted by this defect as well.   

Also, I'm new with these controls so if users like @Alyyc85  who have a work around can quickly mock up what they are talking about as a work around, I'd appreciate it.

**Comment by PeterStaev** [comment]
@Celebrim42 how I made it to work is to bind the control with `ngModel` instead of `formControlName`. This way the control works in all cases I have used it. If you have other controls in the form you just set `[ngModelOptions]="{ standalone: true }"` so that angular does not complain :)

**Comment by Alyyc85** [comment]
Here a simple wrapper that checks value and the presence of the iframe of kendo-editor before patching formControl value.
focusOffset is assigned at every new view of kendo-editor but sometimes the iframe is not ready before formControl changes.

So i used writevalue() method of the CVA to check the existence with Promise.resolve() to wait all asyncronous operations before set the value of the control (and iframe is an asyncronous load). 
With the wrapper you can easily bring this behaviour in all your app without rewrite it every time

https://stackblitz.com/edit/angular-xhyamj-twngt8

**Comment by dtopalov** [comment]
Fixed in the latest [dev version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/) of the Editor package.

**Comment by Raisolution** [comment]
Fix available in `v2.0.2` of the `kendo-angular-editor` package.

**Comment by joharzmn** [comment]
I am facing this issue in version: `    "@progress/kendo-angular-editor": "3.1.2"`
